microcosm_pubsub/handlers/uri_handler.py

```diff
@@ -132,15 +140,24 @@ def get_resource(self, message, uri):
         Passes message context.
 
         """
+        if self.resource_cache:
```

Comment:
and if it's a resource that we want to cache (we don't want to cache mutable objects)